rholang/src/test/scala/coop/rchain/rholang/interpreter/MatchTest.scala

```diff
@@ -3,61 +3,66 @@ package coop.rchain.rholang.interpreter
 import cats.{Eval => _, _}
 import cats.data._
 import cats.implicits._
-
 import coop.rchain.models.Channel.ChannelInstance._
 import coop.rchain.models.Connective.ConnectiveInstance._
 import coop.rchain.models.Expr.ExprInstance._
 import coop.rchain.models.Var.VarInstance._
 import coop.rchain.models.Var.WildcardMsg
-import coop.rchain.models._
-import coop.rchain.rholang.syntax.rholang_mercury.Absyn.{
-  Bundle => AbsynBundle,
-  Ground => AbsynGround,
-  Send => AbsynSend,
-  _
-}
-
+import coop.rchain.models.{EVar, _}
+import coop.rchain.models.rholang.sort.Sortable
+import coop.rchain.rholang.syntax.rholang_mercury.Absyn.{Bundle => AbsynBundle, Ground => AbsynGround, Send => AbsynSend, _}
 import org.scalatest._
 
 import scala.collection.immutable.BitSet
 
 class VarMatcherSpec extends FlatSpec with Matchers {
   import coop.rchain.models.rholang.implicits._
   import SpatialMatcher._
+
+  def assertSpatialMatch[T: Sortable, P: Sortable](
+      target: T,
+      pattern: P,
+      expected: Option[FreeMap])(implicit sm: SpatialMatcher[T, P]): Assertion = {
+    assertSorted(target, "target")
+    assertSorted(pattern, "pattern")
+    expected.foreach(_.values.foreach((v: Par) => assertSorted(v, "expected captured term")))
```

Comment:
> Won't two previous assertions already test that?

No, because the expected captures are provided by hand separately, and we can err in there by providing unsorted terms just as well.

In fact, all the changes I've done to tests were guided by one of those three assertions failing and you can see in the diff that some of the expected captures had to change too. If I didn't put this assertion here, those tests would be enforcing unsorted output for sorted input.

See matcher test line 65